### PR TITLE
ADD: monitor-lodestar & erigon in molecule test

### PR DESCRIPTION
**Pull request by gbayasgalan**
used dashboards - lodestar: https://github.com/ChainSafe/lodestar/tree/unstable/dashboards
erigon: https://gist.github.com/tzumby/46561917c225843e1e1e34b356158085

### CODE REVIEW

File: .github/workflows/test-molecule.yml [hunk 1]
```diff
@@ -309,7 +311,7 @@ jobs:
           # {role: "manage-service", test: "mevboost-besu-nimbus"},
           # {role: "manage-service", test: "mevboost-besu-prysm"},
           # {role: "manage-service", test: "mevboost-besu-teku"},
-          # {role: "manage-service", test: "mevboost-geth-lighthouse"},
+          # {role: "manage-service", test: "mevboost-geth-lodestar"},
```
Comment (stefa2k) [inline .github/workflows/test-molecule.yml:314]: I'd suggest to add `mevboost-geth-lodestar` while still leaving `mevboost-geth-lighthouse` in place (not replace it).

**Review by stefa2k — CHANGES_REQUESTED** [review]
`controls/roles/manage-service/files/grafana/LodestarBeaconService/dashboards/odestar_block_processor.json` I guess this file should be `controls/roles/manage-service/files/grafana/LodestarBeaconService/dashboards/lodestar_block_processor.json` right?

File: .github/workflows/test-molecule.yml (hunk 1 above)
Comment (gbayasgalan) [reply .github/workflows/test-molecule.yml:315]: done!

**Comment by gbayasgalan** [comment]
> `controls/roles/manage-service/files/grafana/LodestarBeaconService/dashboards/odestar_block_processor.json` I guess this file should be `controls/roles/manage-service/files/grafana/LodestarBeaconService/dashboards/lodestar_block_processor.json` right?

yes you are right, that was typo 👍

**Review by stefa2k — APPROVED** [review]
lgtm

**Pull request merged**